pylib/anki/lang.py

```diff
@@ -182,7 +183,12 @@ def get_def_lang(lang: str | None = None) -> tuple[int, str]:
     """Return lang converted to name used on disk and its index, defaulting to system language
     or English if not available."""
     try:
-        (sys_lang, enc) = locale.getdefaultlocale()
+        # getdefaultlocale() is deprecated since Python 3.11 but we need to keep using it due to a bug: https://bugs.python.org/issue38805
+        with warnings.catch_warnings():
+            warnings.simplefilter("ignore", DeprecationWarning)
+            (sys_lang, enc) = locale.getdefaultlocale()
+    except AttributeError:
+        (sys_lang, enc) = locale.getlocale()
```

Comment:
```suggestion
        # this will return a different format on Windows (e.g. Italian_Italy), resulting in us falling back to en_US
        # further below
        (sys_lang, enc) = locale.getlocale()
```